### PR TITLE
Feature/update template

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* #132 

## Main Changes:
* Updated to the newest [EPA template](https://www.epa.gov/themes/epa_theme/pattern-lab/?p=pages-standalone-template), caching all required external files and removing inapplicable code.

## Steps To Test:
1. Test all pages locally (`index.html`, `400.html`, `500.html`, `api_key_signup.html`, & `api-docs/index.html`), and verify everything functions correctly in the context of the new template.

### CODE REVIEW

**Comment by maxdiebold-erg** [comment]
I need to address a font issue before I missed, then I will reopen this.

**Review by cschwinderg — CHANGES_REQUESTED** [review]
The header and footer looks good. 

I'm seeing some style changes in other areas (screenshots below left = new and right = production).

* The results panel now takes up the full width and the R Factor value looks a little different. This one is might be ok though. @coobr01 What is your opinion on this one? Personally, I like that the new version takes up the full width.
image

* The api docs page now has grids around the table cells. I think this one needs to be fixed.
image

File: app/app/public/400.html [hunk 1]
```diff
@@ -1,237 +1,850 @@
...
+          <article class="article">
+            <h1 class="page-title">Sorry, but this web page does not exist.</h1>
+            <div class="main-column">
+              <h2>We want to help you find what you are looking for.</h2>
+
+              <ul>
+                <li>
+                  <a href="https://www.epa.gov/npdes"
+                    >Browser other NPDES related topics</a
+                  >
```
[the remark this answers is not in the document]
Comment (maxdiebold-erg) [reply app/app/public/400.html:557]: I think the one on the right is `500.html` and the left is `400.html`? I'd just copied the content over for both.

**Comment by maxdiebold-erg** [comment]
@cschwinderg Good catches, I'll fix the styles on the API documentation page.

File: app/app/public/400.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (cschwinderg) [reply app/app/public/400.html:557]: You are correct. Thanks for catching that.

**Review by cschwinderg — CHANGES_REQUESTED** [review]
I found one more issue. The responses table has a different header styles. 

Production also has a white box around `Media type` dropdown, but I think our new version looks better. It was also probably just a conflict with the old epa template.

Left=new | right=production
image

**Comment by maxdiebold-erg** [comment]
That last issue should be fixed, and the R factor number should look closer to production's (the number's color is a bit different, as is the surrounding shaded box: I thought it would be more consistent to stick with theme colors).

**Review by cschwinderg — APPROVED** [review]
Looks good! Thanks for making those changes!

**Pull request merged**